Lib/idlelib/idle_test/test_run.py

```diff
@@ -72,7 +72,7 @@ class PseudeInputFilesTest(unittest.TestCase):
 
     def test_misc(self):
         shell = MockShell()
-        f = run.PseudoInputFile(shell, 'stdin', 'utf-8')
+        f = run.StdInFile(shell, 'stdin', 'utf-8')
         self.assertIsInstance(f, io.TextIOBase)
         self.assertEqual(f.encoding, 'utf-8')
         self.assertIsNone(f.errors)
```

Comment:
```suggestion
        self.assertEqual(f.errors, 'strict')
```
Update test and fix CI failures.